java/crossbuild/build-linux-centos.sh

```diff
@@ -26,6 +26,6 @@ export JAVA_HOME=/usr/lib/jvm/java-1.7.0
 # build rocksdb
 cd /rocksdb
 scl enable devtoolset-2 'make jclean clean'
-scl enable devtoolset-2 'PORTABLE=1 make -j8 rocksdbjavastatic'
+scl enable devtoolset-2 'PORTABLE=1 make rocksdbjavastatic'
```

Comment:
Any particular reasons for the changes on the `-j` flag?

Reply:
The build was unstable and failed randomly with unrelated dependency problem, so I removed it, as Adam also advised.